src/index.html

```diff
@@ -17,6 +17,92 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO Logo"
+        />
+      </a>
+      <nav class="navi-bar">
+        <ul class="navi-menu">
+          <li class="navi-item">
+            <a
+              href="index.html"
+              class="navi-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="index.html"
+              class="navi-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="index.html"
+              class="navi-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="index.html"
+              class="navi-link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="index.html"
+              class="navi-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="index.html"
+              class="navi-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="index.html"
+              class="navi-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="navi-item">
+            <a
+              href="index.html"
+              class="navi-link"
+            >
+              Video
+            </a>
```

Comment:
The class names 'navi-item' and 'navi-link' should be updated to follow the BEM naming convention as mentioned earlier.